src/screens/MakeClipScreen.tsx

```diff
@@ -77,7 +77,11 @@ export class MakeClipScreen extends React.Component<Props, State> {
       await AsyncStorage.setItem(PV.Keys.MAKE_CLIP_HOW_TO_HAS_LOADED, JSON.stringify(true))
     }
 
-    const isPublic = await AsyncStorage.getItem(PV.Keys.MAKE_CLIP_IS_PUBLIC)
+    const isPublicString = await AsyncStorage.getItem(PV.Keys.MAKE_CLIP_IS_PUBLIC)
+    let isPublic = null
```

Comment:
Setting this initially to false gets rid of needing to check if its null further down below on line 89. Generally settings something to null means it can potentially be an empty value. I don't think this is the case with this variable. it is either going to be true or false...